src/backend/commands/player/troll/set_bounty.cpp

```diff
@@ -11,9 +11,9 @@ namespace big
 
 		virtual void execute(player_ptr player, const std::vector<std::uint64_t>& _args, const std::shared_ptr<command_context> ctx)
 		{
-			troll::set_bounty_on_player(player, 10000);
+			troll::set_bounty_on_player(player, 10000, true);
```

Comment:
Why this doesn't use `g.session.anonymous_bounty`?

```suggestion
			troll::set_bounty_on_player(player, 10000, g.session.anonymous_bounty);
```